### ISSUE TITLE
Pre-work Checklist: Developer: Bitian

### ISSUE THREAD

**Issue by bzzz-coding** [hackforla/website#3784]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] #3794
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by bzzz-coding** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
- 3 hours (including exploring the repo and job board)

**Comment by bzzz-coding** [comment]
Weekly Update: 

1. Progress - In the past week, I submitted three pull requests: first-issue, second-issue, and a small sized issue. The first two were approved and merged. The third PR was approved and is now waiting to be merged.
2. Blockers - None. Once the PR for the small issue gets merged, I'll start working on a medium sized issue.
3. Availability - 6 hours a week.
4. ETA

**Comment by bzzz-coding** [comment]
Progress: My PR got merged earlier this week so I'll start working on my next issue.
Blockers: No, but I'm paying attention to the discussion on the testing results for issue #3109 
Availability: I'll budget 4-5 hours a week for my next issue.
ETA: I've never worked on a Large issue before so I'll have to first take a look at the project.

**Comment by bzzz-coding** [comment]
- Progress: Working on my first large issue #3341 and trying to understand the GHA code.
Blockers: No blocker yet.
Availability: Tuesdays and Thursdays in the evening.
ETA: I've never worked on a Large issue before so I'll have to first take a look at the project. Right now I'm estimating by 1/28/23.

**Comment by ExperimentsInHonesty** [comment]
@bzzz-coding Good work on the progress report in your issue
- https://github.com/hackforla/website/issues/3341#issuecomment-1435449155
I think you got the hang of things now.  I am going to close this prework issue.

**Comment by HackforLABot** [comment]
## Activity Log: bzzz-coding
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- bzzz-coding opened: https://github.com/hackforla/website/issues/3784 at 01/03/2023, 08:47 PM PST
- bzzz-coding assigned: https://github.com/hackforla/website/issues/3784 at 01/04/2023, 09:23 PM PST
- bzzz-coding assigned: https://github.com/hackforla/website/issues/2874#issuecomment-1049006759 at 01/04/2023, 11:05 PM PST
- bzzz-coding opened: https://github.com/hackforla/website/pull/3795 at 01/05/2023, 12:00 AM PST
- bzzz-coding PR was merged: https://github.com/hackforla/website/pull/3795#event-8182898918 at 01/09/2023, 07:21 AM PST
- bzzz-coding issue closed by PR 3795: https://github.com/hackforla/website/issues/2874#event-8182899276 at 01/09/2023, 07:21 AM PST
- bzzz-coding assigned: https://github.com/hackforla/website/issues/2278#issuecomment-926948584 at 01/09/2023, 11:47 AM PST
- bzzz-coding commented on: https://github.com/hackforla/website/issues/2278#issuecomment-1376199180 at 01/09/2023, 11:53 AM PST
- bzzz-coding opened: https://github.com/hackforla/website/pull/3800 at 01/09/2023, 12:19 PM PST
- bzzz-coding PR was merged: https://github.com/hackforla/website/pull/3800#event-8186990702 at 01/09/2023, 04:11 PM PST
- bzzz-coding issue closed by PR 3800: https://github.com/hackforla/website/issues/2278#event-8186990799 at 01/09/2023, 04:11 PM PST
- bzzz-coding assigned: https://github.com/hackforla/website/issues/3755#issuecomment-1351734938 at 01/10/2023, 10:42 AM PST
- bzzz-coding commented on: https://github.com/hackforla/website/issues/3755#issuecomment-1377696925 at 01/10/2023, 10:45 AM PST
- bzzz-coding commented on: https://github.com/hackforla/website/issues/3784#issuecomment-1377698703 at 01/10/2023, 10:47 AM PST
- bzzz-coding opened: https://github.com/hackforla/website/pull/3806 at 01/10/2023, 01:09 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/3804#issuecomment-1377891189 at 01/10/2023, 01:26 PM PST
- bzzz-coding submitted review: https://github.com/hackforla/website/pull/3804#pullrequestreview-1243009340 at 01/10/2023, 01:39 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/issues/3784#issuecomment-1379432445 at 01/11/2023, 12:16 PM PST
- bzzz-coding PR was merged: https://github.com/hackforla/website/pull/3806#event-8210903191 at 01/12/2023, 04:34 AM PST
- bzzz-coding issue closed by PR 3806: https://github.com/hackforla/website/issues/3755#event-8210903395 at 01/12/2023, 04:34 AM PST
- bzzz-coding assigned: https://github.com/hackforla/website/issues/3109#issuecomment-1321202216 at 01/12/2023, 02:05 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/issues/3109#issuecomment-1381047079 at 01/12/2023, 02:08 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/issues/3109#issuecomment-1381164332 at 01/12/2023, 04:46 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/issues/3109#issuecomment-1381208287 at 01/12/2023, 05:49 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/3811#issuecomment-1381215399 at 01/12/2023, 05:59 PM PST
- bzzz-coding submitted review: https://github.com/hackforla/website/pull/3811#pullrequestreview-1246812834 at 01/12/2023, 06:11 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/issues/3109#issuecomment-1382321028 at 01/13/2023, 11:50 AM PST
- bzzz-coding submitted review: https://github.com/hackforla/website/pull/3811#pullrequestreview-1252079121 at 01/17/2023, 09:19 AM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/3811#issuecomment-1385767530 at 01/17/2023, 09:21 AM PST
- bzzz-coding opened: https://github.com/hackforla/website/pull/3839 at 01/17/2023, 08:51 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/3839#issuecomment-1386488681 at 01/17/2023, 08:57 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/3841#issuecomment-1397417205 at 01/19/2023, 10:20 AM PST
- bzzz-coding submitted review: https://github.com/hackforla/website/pull/3841#pullrequestreview-1262366952 at 01/19/2023, 10:24 AM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/3839#issuecomment-1399379706 at 01/21/2023, 05:33 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/3839#issuecomment-1400703596 at 01/23/2023, 09:18 AM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/3839#issuecomment-1403084599 at 01/24/2023, 08:30 PM PST
- bzzz-coding PR was merged: https://github.com/hackforla/website/pull/3839#event-8414651432 at 02/01/2023, 01:06 PM PST
- bzzz-coding issue closed by PR 3839: https://github.com/hackforla/website/issues/3109#event-8414651640 at 02/01/2023, 01:06 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/3924#issuecomment-1414407651 at 02/02/2023, 01:37 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/issues/3784#issuecomment-1414419681 at 02/02/2023, 01:47 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/3926#issuecomment-1414424089 at 02/02/2023, 01:52 PM PST
- bzzz-coding submitted review: https://github.com/hackforla/website/pull/3926#pullrequestreview-1282056948 at 02/02/2023, 03:19 PM PST
- bzzz-coding submitted review: https://github.com/hackforla/website/pull/3924#pullrequestreview-1282070221 at 02/02/2023, 03:38 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/3923#issuecomment-1414525089 at 02/02/2023, 03:56 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/3927#issuecomment-1414588606 at 02/02/2023, 05:28 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/3927#issuecomment-1420059286 at 02/06/2023, 06:00 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/3923#issuecomment-1420062357 at 02/06/2023, 06:02 PM PST
- bzzz-coding submitted review: https://github.com/hackforla/website/pull/3927#pullrequestreview-1286291801 at 02/06/2023, 06:21 PM PST
- bzzz-coding submitted review: https://github.com/hackforla/website/pull/3923#pullrequestreview-1286309729 at 02/06/2023, 06:51 PM PST
- bzzz-coding assigned: https://github.com/hackforla/website/issues/3341#issuecomment-1399385526 at 02/06/2023, 07:09 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/issues/3341#issuecomment-1420135317 at 02/06/2023, 07:12 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/3927#issuecomment-1421293131 at 02/07/2023, 10:58 AM PST
- bzzz-coding commented on: https://github.com/hackforla/website/issues/3784#issuecomment-1426365719 at 02/10/2023, 01:27 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/issues/3341#issuecomment-1435449155 at 02/17/2023, 05:58 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/4017#issuecomment-1440531851 at 02/22/2023, 10:08 AM PST
- bzzz-coding submitted review: https://github.com/hackforla/website/pull/4017#pullrequestreview-1310459370 at 02/22/2023, 04:10 PM PST
- bzzz-coding submitted review: https://github.com/hackforla/website/pull/4017#pullrequestreview-1314375666 at 02/24/2023, 03:23 PM PST
- bzzz-coding issue closed as completed: https://github.com/hackforla/website/issues/3784#event-8609001402 at 02/25/2023, 10:32 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/4017#issuecomment-1449285028 at 02/28/2023, 07:40 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/issues/3341#issuecomment-1454355593 at 03/03/2023, 07:17 PM PST
- bzzz-coding opened: https://github.com/hackforla/website/pull/4109 at 03/03/2023, 11:51 PM PST
- bzzz-coding closed: https://github.com/hackforla/website/pull/4109#event-8665566196 at 03/04/2023, 12:34 AM PST
- bzzz-coding opened: https://github.com/hackforla/website/pull/4110 at 03/04/2023, 09:53 AM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/4110#issuecomment-1454825805 at 03/04/2023, 10:06 AM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/4110#issuecomment-1455009034 at 03/04/2023, 11:04 PM PST
- bzzz-coding submitted review: https://github.com/hackforla/website/pull/4017#pullrequestreview-1329566576 at 03/07/2023, 01:30 PM PST
- bzzz-coding commented on: https://github.com/hackforla/website/pull/4156#issuecomment-1467086638 at 03/13/2023, 03:55 PM PDT
- bzzz-coding commented on: https://github.com/hackforla/website/pull/4110#issuecomment-1467160800 at 03/13/2023, 05:23 PM PDT
- bzzz-coding submitted review: https://github.com/hackforla/website/pull/4156#pullrequestreview-1340045873 at 03/14/2023, 11:32 AM PDT
- bzzz-coding commented on: https://github.com/hackforla/website/pull/4176#issuecomment-1469240775 at 03/14/2023, 08:05 PM PDT
- bzzz-coding submitted review: https://github.com/hackforla/website/pull/4176#pullrequestreview-1342488709 at 03/15/2023, 02:56 PM PDT
- bzzz-coding PR was merged: https://github.com/hackforla/website/pull/4110#event-8786486927 at 03/18/2023, 04:47 PM PDT
- bzzz-coding issue closed by PR 4110: https://github.com/hackforla/website/issues/3341#event-8786487020 at 03/18/2023, 04:47 PM PDT
- bzzz-coding assigned: https://github.com/hackforla/website/issues/4158 at 03/23/2023, 09:10 PM PDT
- bzzz-coding commented on: https://github.com/hackforla/website/issues/4158#issuecomment-1482224065 at 03/23/2023, 09:12 PM PDT
- bzzz-coding unassigned: https://github.com/hackforla/website/issues/4158 at 03/28/2023, 07:24 PM PDT
- bzzz-coding commented on: https://github.com/hackforla/website/pull/4338#issuecomment-1488938084 at 03/29/2023, 09:38 AM PDT
- bzzz-coding submitted review: https://github.com/hackforla/website/pull/4338#pullrequestreview-1363800564 at 03/29/2023, 12:22 PM PDT
- bzzz-coding merged: https://github.com/hackforla/website/pull/4260#event-8880087206 at 03/29/2023, 12:38 PM PDT
- bzzz-coding closed by PR 4260: https://github.com/hackforla/website/issues/3860#event-8880087437 at 03/29/2023, 12:39 PM PDT
- bzzz-coding merged: https://github.com/hackforla/website/pull/4338#event-8889933069 at 03/30/2023, 10:04 AM PDT
- bzzz-coding commented on: https://github.com/hackforla/website/pull/4280#issuecomment-1493686693 at 04/02/2023, 10:29 PM PDT
- bzzz-coding commented on: https://github.com/hackforla/website/pull/4339#issuecomment-1497841175 at 04/05/2023, 10:07 AM PDT
- bzzz-coding commented on: https://github.com/hackforla/website/issues/2652#issuecomment-1498004446 at 04/05/2023, 12:25 PM PDT
- bzzz-coding commented on: https://github.com/hackforla/website/pull/4339#issuecomment-1498022904 at 04/05/2023, 12:42 PM PDT
- bzzz-coding commented on: https://github.com/hackforla/website/pull/4280#issuecomment-1498036793 at 04/05/2023, 12:54 PM PDT
- bzzz-coding submitted review: https://github.com/hackforla/website/pull/4339#pullrequestreview-1373705403 at 04/05/2023, 02:15 PM PDT
- bzzz-coding commented on: https://github.com/hackforla/website/pull/4280#issuecomment-1498483038 at 04/05/2023, 09:39 PM PDT
- bzzz-coding commented on: https://github.com/hackforla/website/pull/4280#issuecomment-1499415338 at 04/06/2023, 10:46 AM PDT
- bzzz-coding commented on: https://github.com/hackforla/website/pull/4280#issuecomment-1499638391 at 04/06/2023, 02:13 PM PDT
- bzzz-coding commented on: https://github.com/hackforla/website/pull/4280#issuecomment-1499661927 at 04/06/2023, 02:42 PM PDT
- bzzz-coding submitted review: https://github.com/hackforla/website/pull/4280#pullrequestreview-1375684165 at 04/06/2023, 04:34 PM PDT
- bzzz-coding commented on: https://github.com/hackforla/website/issues/2652#issuecomment-1504284051 at 04/11/2023, 04:50 PM PDT
